src/helpers/eventBridge.js

```diff
@@ -87,6 +92,8 @@ export default class EventBridge {
       })
       .promise();
 
+    await this.getEvents(); // need to clear this manual published event from the SQS observer queue.
```

Comment:
What's the distinction between manual and non-manual published events here?

Reply:
when we trigger an event to be published and then assert it is published it's cleared from SQS, when we use sls-test-tools to publish an event we make no assertion, and therefore it is not removed. 
We will likely need to have cleanup between tests soon, and make use of a FIFO SQS in the future...